tests/pytest/test_mem.py

```diff
@@ -0,0 +1,78 @@
+
+from common import *
+import time
+import datetime
+
+
+CHECK_MEMRECLAIM = os.getenv('MEMRECLAIM', '0') == '1'
+
+JSON_FILES = [
```

Comment:
@rafie I think it's better to copy these files so it won't change without us knowing about it.

Reply:
They can become quite big and I don't want to introduce git-lfs. When it breaks, we'll fix it.